common/test/acceptance/pages/studio/settings.py

```diff
@@ -208,6 +238,17 @@ def wait_for_prerequisite_course_options(self):
     # Clicks
     ################
 
+    def click_button(self, name):
+        """
+        Clicks the button
+        """
+        btn_css = 'div#page-notification button.action-%s' % name.lower()
```

Comment:
Use the format method rather than string interpolation with the % sign. Like this (not checked for syntax):
`btn_css = 'div#page-notification button.action-{}'.format(name.lower())`